app/javascript/dashboard/routes/dashboard/settings/inbox/Settings.vue

```diff
@@ -173,7 +191,9 @@ export default {
     },
     canLocktoSingleConversation() {
       return (
-        this.isASmsInbox || this.isAWhatsAppChannel || this.isAFacebookInbox
+        isASmsInbox(this.inbox) ||
+        isAWhatsAppChannel(this.inbox) ||
+        isAFacebookInbox(this.inbox)
       );
     },
     inboxNameLabel() {
```

Comment:
In this computed property, we are not using the `isAWebWidgetInbox` helper. 
And same for this `inboxNamePlaceHolder` computed property.